content/applications/productivity/iot/config/troubleshooting.rst

```diff
@@ -387,6 +387,26 @@ Devices --> Barcode Device`) and activating the :guilabel:`Is scanner` option.
    :align: center
    :alt: Modifying the form view of the barcode scanner.
 
+Barcode scanner processes barcode characters individually
+---------------------------------------------------------
+
+When accessing the mobile version of Odoo from a mobile device, or tablet, paired with a barcode
+scanner, via the :abbr:`IoT (Internet of Things)` box, the scanner may process each barcode
+character as an individual scan. In this case, the :guilabel:`Keyboard Layout` option **must** be
+filled out with the appropriate language of the barcode scanner on the :guilabel:`Barcode Scanner`
+form page.
+
+.. tip::
+   Access the barcode scanner by navigating to :menuselection:`IoT App --> Devices --> Barcode
+   Scanner`.
+
+.. image:: troubleshooting/keyboard-layout.png
+   :align: center
+   :alt: Barcode scanner form page, with keyboard layout option highlighted.
+
+The keyboard layout is language based, and the options available vary, depending on the device and
+the language of the database. For example: `English (UK)`, `English (US)`, etc.
```

Comment:
Because the active instruction to navigate to the barcode scanner form page is _after_ stating the option that needs to be enabled, the option's text is not currently present on the UI, so no guilabel. However, after this instruction, :guilabel: can be used.
```suggestion
character as an individual scan. In this case, the *Keyboard Layout* option **must** be filled out
with the appropriate language of the barcode scanner on the *Barcode Scanner* form page.

.. tip::
   Access the barcode scanner form page by navigating to :menuselection:`IoT App --> Devices -->
   Barcode Scanner`.

.. image:: troubleshooting/keyboard-layout.png
   :align: center
   :alt: Barcode scanner form page, with keyboard layout option highlighted.

The :guilabel:`Keyboard Layout` is language based, and the options available vary, depending on the
device and the language of the database. For example: :guilabel:`English (UK)`, :guilabel:`English
(US)`, etc.
```